tests/smoke/test_ucc_build.py

```diff
@@ -1,6 +1,7 @@
 import difflib
 import tempfile
 from os import path
+from pathlib import Path
```

Comment:
Let's be consistent here with how we treat path, can you please use just `os.path`?